package-lock.json

```diff
@@ -1,23 +1,4012 @@
 {
   "name": "tech-doc-hugo",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Is this due to a different npm version or something?
Seems a bit weird to have a diff with twice as many additions as removals as a result of removing a dependency.

Reply:
(Sorry Richard, I didn't spot your comment before lgtm-ing)

Reply:
My process was to remove alex from `packages.json` and then running `./scripts/verify-lint` which calls `./scripts/update-node_modules`, which caused the huge change to `package-lock.json`. I'm not a node expert by any means, but this kind of thing seems par-for-the-course when it comes to node.

As for why the lockfile version changed, I don't really know! My local npm is v7.20.5 (nodejs v16.6.2) and it just... upgraded the number.

I figured since the tests passed it was probably fine to upgrade that lockfileVersion number.